From: Robert Schwebel <r.schwebel@pengutronix.de>
Subject: [PATCH] fix blkid.h include

When building util-linux-ng-2.15, I get this:

make[3]: Entering directory `/ptx/work/octopus/WORK_2_B/rsc/svn/oselas/bsp/OSELAS.BSP-Pengutronix-AllYes-trunk/platform-phyCORE-PXA270.PCM990/build-target/util-linux-ng-2.15/mount'
arm-iwmmx-linux-gnueabi-gcc -std=gnu99 -DHAVE_CONFIG_H -I. -I..  -include ../config.h -I../include -DLOCALEDIR=\"/usr/share/locale\" -isystem /home/rsc/svn/oselas/bsp/OSELAS.BSP-Pengutronix-AllYes-trunk/platform-phyCORE-PXA270.PCM990/sysroot-target/include -isystem /home/rsc/svn/oselas/bsp/OSELAS.BSP-Pengutronix-AllYes-trunk/platform-phyCORE-PXA270.PCM990/sysroot-target/usr/include  -fsigned-char  -I/home/rsc/svn/oselas/bsp/OSELAS.BSP-Pengutronix-AllYes-trunk/platform-phyCORE-PXA270.PCM990/sysroot-target/usr/include    -g -O2 -MT mount-fsprobe.o -MD -MP -MF .deps/mount-fsprobe.Tpo -c -o mount-fsprobe.o `test -f '../lib/fsprobe.c' || echo './'`../lib/fsprobe.c
../lib/fsprobe.c:10:19: error: blkid.h: No such file or directory

The reason is that blkid.h comes from e2fsutils, which install the header file
into /usr/include/blkid/blkid.h; /usr/include is also what the .pc file
specifies for the include path for the CFLAGS. So IMHO the patch below must be
applied to make it work.

If this is not acceptable, a little bit more configure magic would be necessary.

Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>

---
 lib/fsprobe.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: util-linux-ng-2.15/lib/fsprobe.c
===================================================================
--- util-linux-ng-2.15.orig/lib/fsprobe.c
+++ util-linux-ng-2.15/lib/fsprobe.c
@@ -7,7 +7,7 @@
 #include <sys/types.h>
 #include <stdlib.h>
 
-#include <blkid.h>
+#include <blkid/blkid.h>
 
 #include "blkdev.h"
 #include "canonicalize.h"
